src/status_im/chat/views/message/message.cljs

```diff
@@ -111,13 +114,39 @@
        ffirst
        replacements))
 
+(defn- parse-url [string on-press]
```

Comment:
This looks like a good candidate for unit testing!
Also could you simplify it a bit by introducing some private helper functions?

Reply:
I agree that unit tests would be super nice for that function !